test/util/kibishii/kibishii_utils.go

```diff
@@ -58,6 +58,11 @@ var KibishiiStorageClassName = "kibishii-storage-class"
 // RunKibishiiTests runs kibishii tests on the provider.
 func RunKibishiiTests(veleroCfg VeleroConfig, backupName, restoreName, backupLocation, kibishiiNamespace string,
 	useVolumeSnapshots, defaultVolumesToFsBackup bool) error {
+	if veleroCfg.CloudProvider == "kind" {
+		// don't test snapshots or filesystem backups on kind
+		useVolumeSnapshots = false
+		defaultVolumesToFsBackup = false
+	}
```

Comment:
Kibishii workload is for PV data B/R test purpose, so it's better to skip tests which are consuming `RunKibishiiTests` using your new label`[SKIP_KIND]`.

Reply:
I might add namespacemapping for kind then which skips restic/snapshot

Reply:
Alternative to RunKibishiiTests for stateless workloads that can run on KinD

Reply:
as a follow up after rebasing against https://github.com/vmware-tanzu/velero/pull/7297